.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -275,7 +280,7 @@ jobs:
       - init-github-context
       - rawls-swat-e2e-test-job
       - delete-billing-project-v2-from-bee-workflow
-    if: ${{ needs.init-github-context.outputs.delete-bee && always() }} # always run to confirm bee is destroyed unless explicitly requested not to
+    if: ${{ needs.init-github-context.outputs.delete-bee == true || needs.init-github-context.outputs.delete-bee == 'true' }}
```

Comment:
I don't think the `delete-bee` option was working properly before given the stringification of booleans.